src/appshell/qml/AboutDialog.qml

```diff
@@ -97,23 +97,27 @@ QmlDialog {
                 StyledTextLabel {
                     Layout.fillWidth: true
                     text: {
-                        var message = qsTrc("appshell", "Visit <a href='%1'>%2</a> for new versions and more information.<br>Support MuseScore with your <a href='%3'>%4</a>.")
+                        var message = qsTrc("appshell", "Visit %1 for new versions and more information.<br>Get %2 with the program or %3 to is development.")
```

Comment:
Trying to avoid as much HTML code as possible, to make translations easier and more secure too.
The `<br>` though is needed here, not sure how well that works with translations, in 3.x there was a `\n` that for replaces by a `<br>` after translation did its job.

Reply:
I just found the solution for the `<br>` issue. It is actually identical to what 3.x used, just in qml rather than c++.